attachment_queue/__manifest__.py

```diff
@@ -5,7 +5,7 @@
     "name": "Attachment Queue",
     "version": "12.0.1.0.0",
     "author": "Akretion,Odoo Community Association (OCA)",
-    "summary": "Base module that add the concept of queue for processing file",
+    "summary": "Base module that adding the concept of queue for processing file",
```

Comment:
s/that adding/adding/

Reply:
why don't use suggestion here ?

Reply:
not sure what you meant. I suggested using "adding" not "that adding" which is not correct.

Reply:
ahhhh the Github suggestions... Sorry @bealdav I didn't know the feature. Seb just explained me...